cluster/manifests/nvidia/nvidia-driver-installer.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nvidia-driver-installer
+  namespace: kube-system
+  labels:
+    application: nvidia-driver-installer
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ''
+spec:
+  selector:
+    matchLabels:
+      application: nvidia-driver-installer
+  template:
+    metadata:
+      labels:
+        application: nvidia-driver-installer
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
I think this annotation is deprecated

Reply:
@muaazsaleem https://github.com/zalando-incubator/kubernetes-on-aws/pull/974